pkg/controllers/vdb/init_db_test.go

```diff
@@ -147,10 +147,13 @@ var _ = Describe("init_db", func() {
 		res, err := g.ConstructConfigParms(ctx)
 		ExpectWithOffset(1, err).Should(Succeed())
 		ExpectWithOffset(1, res).Should(Equal(ctrl.Result{}))
-		Expect(g.ConfigurationParams.Size()).Should(Equal(1))
+		Expect(g.ConfigurationParams.Size()).Should(Equal(2))
 		v, ok := g.ConfigurationParams.Get("InitialDefaultSubclusterName")
 		Expect(ok).Should(BeTrue())
 		Expect(v).Should(Equal(vdb.Spec.Subclusters[0].Name))
+		v, ok = g.ConfigurationParams.Get("EncryptSpreadComm")
```

Comment:
I think you can create a new test function for spread encryption where you can directly call `setEncryptSpreadCommConfigIfNecessary()` and have a few scenarios by changing `EncryptSpreadComm` and vertica version.